eggplant_project/settings/base.py

```diff
@@ -15,6 +15,10 @@
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.messages import constants as messages_constants
 
+COOP_NAME = 'KBHFF'
```

Comment:
I already created the issue, @pawciobiel, but thanks for your understanding and collaboration :)

And yes, it is a design pattern... but I wouldn't go as far as to call it conventional, because too many places there's no design patterns or only anti-patterns.